proof/refine/ARM/Reply_R.thy

```diff
@@ -96,4 +95,487 @@ lemma replyRemoveTCB_st_tcb_at'_sym_ref:
   apply (clarsimp simp: obj_at'_def pred_tcb_at'_def)
   done
 
+lemma getReplyTCB_wp:
+  "\<lbrace>\<lambda>s. reply_at' rptr s \<longrightarrow> obj_at' (\<lambda>reply. P (replyTCB reply) s) rptr s\<rbrace>
+   getReplyTCB rptr
+   \<lbrace>P\<rbrace>"
```

Comment:
This slightly clashes with the version of this lemma I added in my PR. See https://github.com/seL4/l4v/pull/93/files#diff-8f12378cc553b81307fa6f31bbc1b337R2395

Reply:
Updated to use yours.